Formula/cryptominisat.rb

```diff
@@ -0,0 +1,37 @@
+class Cryptominisat < Formula
+  desc "Advanced SAT solver"
+  homepage "https://www.msoos.org/"
+  url "https://github.com/msoos/cryptominisat/archive/5.6.6.tar.gz"
+  sha256 "95fac3df4311d4fb6e2674b1bce3113056795a2eda51cd807f73bcc4f9b1a2d5"
+
+  depends_on "cmake" => :build
+  depends_on :arch => :x86_64
+  depends_on "boost"
+  depends_on "python"
+
+  def install
+    args = ["-DCMAKE_INSTALL_PREFIX=#{prefix}",
+            "-DNOM4RI=ON"]
+
+    if build.without? "python"
+      args << "--DENABLE_PYTHON_INTERFACE=OFF"
+    end
+
+    mkdir "build" do
+      system "cmake", *args, ".."
```

Comment:
`system "cmake", "..", *std_cmake_args, "-DNOM4RI=ON"`
(this will take care of prefix and other stuff)

Reply:
Cool, good to know!